storage/rocksdb/ha_rocksdb.cc

```diff
@@ -8494,6 +8494,23 @@ int ha_rocksdb::index_read_map_impl(uchar *const buf, const uchar *const key,
         kd.get_key_parts()) {
       using_full_key = false;
     }
+
+    if (m_insert_with_update && m_dup_key_found &&
+        active_index == m_dup_key_found) {
```

Comment:
This conditional check looks wrong. active_index is uint and m_dup_key_found is a bool. Is this meant to be m_dupp_errkey?

Reply:
Yes. Good catch. Fixed